seedfarmer/mgmt/module_info.py

```diff
@@ -737,10 +792,33 @@ def remove_deployed_deployment_manifest(deployment: str, session: Optional[Sessi
     ssm.delete_parameters(parameters=[_deployed_deployment_manifest_key(deployment)], session=session)
 
 
+def remove_testmetatainput(deployment: str, group: str, module: str, session: Optional[Session] = None) -> None:
```

Comment:
can we rename this to `remove_test_metadata_input()` and then follow snakecase for the other metadata functions?

Reply:
yes